101-hdinsight-linux-ssh-publickey/azuredeploy.json

```diff
@@ -99,10 +94,10 @@
         "storageProfile": {
           "storageaccounts": [
             {
-              "name": "[concat(parameters('clusterStorageAccountName'),'.blob.core.windows.net')]",
+              "name": "[replace(replace(concat(reference(concat('Microsoft.Storage/storageAccounts/', variables('storageAccountName')), variables('defaultApiVersion')).primaryEndpoints.blob),'https:',''),'/','')]",
```

Comment:
Happy to help! We (the Azure Tools team) are working on changes to standardize other parts of the templates, so we really appreciate your work to update the storage endpoints.